lib/tasks/migrate_old_tinymce_assets.rake

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+# rubocop:disable Metrics/LineLength
+namespace :tinymce_assets do
+  desc 'Migrate old TinyMCE images to new polymorphic format' \
+       'IT SHOULD BE RUN ONE TIME ONLY'
+  task migrate_tinymce_assets: :environment do
+    old_images = TinyMceAsset.where('step_id IS NOT NULL OR result_text_id IS NOT NULL')
```

Comment:
Maybe you could add additional condition, to have the object null, so we're completely safe if this task is (for whatever reason) run more than onece